app/src/main/java/com/dguitarclassic/todoapps/db/TodoDao.kt

```diff
@@ -1,19 +1,22 @@
 package com.dguitarclassic.todoapps.db
 
-import androidx.lifecycle.LiveData
 import androidx.room.Dao
 import androidx.room.Insert
 import androidx.room.OnConflictStrategy
 import androidx.room.Query
 import com.dguitarclassic.todoapps.model.Todo
+import kotlinx.coroutines.flow.Flow
 
 @Dao
 interface TodoDao {
 
     @Query("SELECT * FROM todo ORDER BY id DESC")
-    fun getAllTodo() : LiveData<List<Todo>>
+    fun getAllToDos(): Flow<List<Todo>>
 
     @Insert(onConflict = OnConflictStrategy.IGNORE)
-    fun addTodo(todo: Todo)
+    suspend fun insert(toDo: Todo)
+
+    @Query("DELETE FROM todo WHERE id LIKE :id")
```

Comment:
Why using LIKE in the query?

Reply:
kemaren ada masalah soal versi dari dependencies yang aku tambahin sama versi kotlinnya, pake @delete sebelumnya itu error, terus aku nyoba pake query aja ternyata ga error, tapi emang belum ku implement fungsi delete nya mas

Reply:
Iyaa bay, disini mau pake annotation @Query atau @Delete sebenernya sama aja ga masalah karena sama aja fungsinya, cuma kalo di QUERY yg lu buat itu kurang tepat kalo pake LIKE, better pake simbol '=' aja cukup (WHERE id = :id), kalo LIKE setauku gua itu akan cari yg mirip